portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactToVCardConverterTest.java

```diff
@@ -151,6 +124,47 @@ public void testConversionToVCardNames() {
         .containsExactly(alternateSourceType1, alternateSourceType2);
   }
 
+  @Test
+  public void testConversionToVCardAddress() {
+    // Set up test: person with a primary address and a secondary address
+    String primaryStreet = "221B Baker St";
+    String primaryCity = "London";
+    String primaryPostcode = "NW1";
+    String primaryCountry = "United Kingdom";
+    com.google.api.services.people.v1.model.Address primaryAddress =
+        new com.google.api.services.people.v1.model.Address()
+            .setStreetAddress(primaryStreet).setCity(primaryCity).setPostalCode(primaryPostcode)
+            .setCountry(primaryCountry).setMetadata(PRIMARY_FIELD_METADATA);
+
+    String altStreet = "42 Wallaby Way";
+    String altCity = "Sydney";
+    String altRegion = "New South Wales";
+    String altCountry = "Australia";
+    com.google.api.services.people.v1.model.Address altAddress =
+        new com.google.api.services.people.v1.model.Address()
+            .setStreetAddress(altStreet).setCity(altCity).setRegion(altRegion)
+            .setCountry(altCountry).setMetadata(SECONDARY_FIELD_METADATA);
+
+    Person person = DEFAULT_PERSON.setAddresses(Arrays.asList(altAddress, primaryAddress));
+
+    // Run test
+    VCard vCard = GoogleContactToVCardConverter.convert(person);
+
+    // Check results for correct values and preferences
+    List<Address> actualPrimaryAddressList =
+        getPropertiesWithPreference(vCard, Address.class, VCARD_PRIMARY_PREF);
+    assertThat(actualPrimaryAddressList.size()).isEqualTo(1);
```

Comment:
Can this be remove cause the next statement seems to verify there is only one element?

Reply:
You are correct!